editor/debugger/script_editor_debugger.cpp

```diff
@@ -530,13 +530,24 @@ void ScriptEditorDebugger::_parse_message(const String &p_msg, const Array &p_da
 		error->set_text(1, error_title);
 		tooltip += " " + error_title + "\n";
 
+		// Find the language of the error's source file.
+		String source_language_name = "C++"; // Default value is the old hard-coded one.
+		const String source_file_extension = oe.source_file.get_extension();
+		for (int i = 0; i < ScriptServer::get_language_count(); ++i) {
+			ScriptLanguage *script_language = ScriptServer::get_language(i);
+			if (source_file_extension == script_language->get_extension()) {
+				source_language_name = script_language->get_name();
+				break;
+			}
+		}
+
 		if (!oe.error_descr.is_empty()) {
 			// Add item for C++ error condition.
 			TreeItem *cpp_cond = error_tree->create_item(error);
-			cpp_cond->set_text(0, "<" + TTR("C++ Error") + ">");
+			cpp_cond->set_text(0, "<" + source_language_name + " " + TTR("Error") + ">");
```

Comment:
This breaks localization, in French it would become "<C++ Erreur>" when it should be "<Erreur C++>" or similar.

You need to use the `%s` substitution to let translators move it around - and likely add a translator comment to clarify the meaning.
Like:
```
# TRANSLATORS: %s is the name of a language, e.g. C++.
cpp_cond->set_text(0, "<" + vformat(TTR("%s Error"), source_language_name) + " " + TTR("Error") + ">");

Same below.